packages/foundation/tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "noImplicitAny": true,
     "moduleResolution": "node",
     "preserveConstEnums": true,
+    "skipLibCheck": true,
```

Comment:
How does this impact the editor?

Reply:
I thought it was needed at one point to get around an API Extractor issue, but turns out it's probably not needed